code/datums/components/plumbing/_plumbing.dm

```diff
@@ -126,9 +123,10 @@
 					direction = "east"
 				if(WEST)
 					direction = "west"
-			I = image('icons/obj/plumbing/plumbers.dmi', "[direction]-[color]", layer = A.layer - 1)
+			I = image('icons/obj/plumbing/plumbers.dmi', "[direction]-[color]", layer = AM.layer - 1)
```

Comment:
```
			I = image('icons/obj/plumbing/plumbers.dmi', "[direction]-[color]", layer = AM.layer - 1, dir = the_direction_you_want)
```
To fix the issue of the overlay defaulting `NORTH`

Reply:
Doing that permanently locks it in that direction. Same for changing it afterwards

Reply:
> Doing that permanently locks it in that direction. Same for changing it afterwards

I think this because of logic that `dir` used in `icon_state`